concord-ui/package.json

```diff
@@ -8,13 +8,14 @@
     "react-intl": "2.4.0",
     "react-redux": "5.0.7",
     "react-router-dom": "4.2.2",
-    "react-scripts": "2.1.1",
+    "react-scripts": "^2.1.8",
     "react-select": "^2.1.2",
     "redux": "4.0.1",
     "redux-thunk": "2.3.0",
     "rmwc": "4.0.6",
     "superagent": "4.0.0",
-    "webfontloader": "1.6.28"
+    "webfontloader": "1.6.28",
+    "@rmwc/list": "latest"
```

Comment:
version should be pinned

Reply:
upgrading rmwc in the next PR, will fix it then